io/slow5/slow5_test.go

```diff
@@ -204,3 +205,31 @@ func TestWrite(t *testing.T) {
 		t.Errorf("Example and test write are different")
 	}
 }
+
+func TestSvb(t *testing.T) {
+	file, _ := os.Open("data/example.slow5")
+	defer file.Close()
+	const maxLineSize = 2 * 32 * 1024
+	parser, _, _ := NewParser(file, maxLineSize)
+	var outputReads []Read
+	for {
+		read, err := parser.ParseNext()
+		if err != nil {
```

Comment:
Please make this an explicit check for `io.EOF`. As written, this test will pass (even though it shouldn't!) if there are any I/O errors while reading from disk.

Reply:
I think we already have that check in `TestParseReads`, but I'll add it here too